packages/axes/src/Axes.ts

```diff
@@ -453,6 +455,68 @@ class Axes extends Component<AxesEvents> {
     return this;
   }
 
+  /**
+   * Change the options of Axes instance.
+   * @ko 인스턴스의 옵션을 변경한다.
+   * @param {AxesOption} options Axes options to change <ko>변경할 옵션 목록</ko>
+   * @return {eg.Axes} An instance of a module itself <ko>모듈 자신의 인스턴스</ko>
+   * @example
+   * ```js
+   * const axes = new eg.Axes({
+   *   "x": {
+   *      range: [0, 100]
+   *   },
+   * }, {
+   *   round: 10,
+   * });
+   *
+   * axes.setTo({"x": 48});
+   * axes.get(); // {"x": 50}
+   *
+   * axes.setOptions({
+   *   round: 1,
+   * });
+   *
+   * axes.setTo({"x": 48});
+   * axes.get(); // {"x": 48}
+   * ```
+   */
+  public setOptions(options: AxesOption) {
+    Object.assign(this.options, options);
```

Comment:
I think this will remain in the build, please check whether it does.
As `Object.assign` isn't supported in IE
- https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Reply:
Also, aren't there any internal updates required for this change?
Check this example of updating options in View360
- https://github.com/naver/egjs-view360/blob/52b16b26a3b6397fef6e7798bfc9874e1a4166a3/src/YawPitchControl/YawPitchControl.ts#L390

Like, `nested` may require some immediate update of event handlers.

Reply:
It looks like this changes should avoid using `Object.assign` for browser compatibility.
Also changing the `nested` option will not require any changes to the event handler.
We are using the following statement to add property to native event.
```ts
    if (!this.options.nested || !this._isEndofAxis(offset, depaPos, destPos)) {
      nativeEvent.__childrenAxesAlreadyChanged = true;
    }
```
Since the option is checked when change is made, the changed option will be applied from the next input without changing any event handlers.

Reply:
If there are parts that need immediate change, it may be current animation being played that duration needs to be changed by minimum/maximum duration option.
Also it will be necessary to discuss about action when the current range is outside of the new range.